python/ray/scripts/scripts.py

```diff
@@ -363,32 +390,70 @@ def start(node_ip_address, redis_address, address, redis_port, port,
           autoscaling_config, no_redirect_worker_output, no_redirect_output,
           plasma_store_socket_name, raylet_socket_name, temp_dir, include_java,
           java_worker_options, load_code_from_local, internal_config,
-          lru_evict, enable_object_reconstruction, metrics_export_port):
+          lru_evict, enable_object_reconstruction, metrics_export_port,
+          log_old_style, log_color, verbose):
     """Start Ray processes manually on the local machine."""
+    cli_logger.old_style = log_old_style
+    cli_logger.color_mode = log_color
+    cli_logger.verbosity = verbose
+
     if gcs_server_port and not head:
         raise ValueError(
             "gcs_server_port can be only assigned when you specify --head.")
 
     if redis_address is not None:
+        cli_logger.abort(
+            "{} is deprecated. Use {} instead.",
+            cf.bold("--redis-address"),
+            cf.bold("--address"))
```

Comment:
does it make sense to not do special formatting for these deprecation messages?

we're going to remove them in a couple weeks anyways